resources/lang/en/general.php

```diff
@@ -39,6 +39,8 @@
     'bulk_delete'           => 'Bulk Delete',
     'bulk_actions'          => 'Bulk Actions',
     'bulk_checkin_delete'   => 'Bulk Checkin Items from Users',
+    'byod'                  => 'BYOD',
```

Comment:
I think I would prefer this label to be: "Bring Your Own Device (BYOD)"

Reply:
That's going to make the table column really wide for just a checkmark or an X though, which is why I chose the shorter version. I think most people in IT know what BYOD means?

Reply:
Okay, I'll allow that.